### PR TITLE
Fixed typo in docs/topics/db/managers.txt.

### PR DESCRIPTION
**Pull request by ebram96**

### CODE REVIEW

File: docs/topics/db/managers.txt [hunk 1]
```diff
@@ -381,7 +381,7 @@ this base class::
             abstract = True
 
 If you use this directly in a subclass, ``objects`` will be the default
-manager if you declare no managers in the base class::
+manager if you declare no managers in the subclass::
```
Comment (shangxiao) [inline docs/topics/db/managers.txt:384]: Nice find 👍  though I just wonder whether we should use "child class" to be consistent with the next example? 🤔

**Review by felixxm — COMMENTED** [review]
@ebram96 Good catch :dart: Thanks :+1:

File: docs/topics/db/managers.txt (hunk 1 above)
Comment (felixxm) [reply docs/topics/db/managers.txt:384]: @ebram96 Can you update your patch? :point_up:

Comment (ebram96) [reply docs/topics/db/managers.txt:384]: Sure, I'm updating it now.

**Review by shangxiao — APPROVED** [review]
🌟 🏆

**Pull request merged**